docs/description/description.json

```diff
@@ -0,0 +1,74 @@
+[
+  {
+    "patternId": "pylint",
+    "title": "Pylint tool",
+    "description": "Pylint is the most comprehensive static analysis tool for Python. It is extremely thorough and is the source of most messages that prospector outputs.",
+    "timeToFix": 5
+  },
+  {
+    "patternId": "pep8",
+    "title": "pep8 tool",
+    "description": "PEP 8 -- Style Guide for Python Code",
```

Comment:
Just a suggestion, but maybe we could be more consistent in the way we start the "description" for each tool? So, for example, this could become:

"pep8 is a tool that validates Python code against the PEP 8 Style Guide."